src/RWLock.h

```diff
@@ -52,15 +52,15 @@ namespace VDMS {
         // Backoff variables.
         // *** Tune experimentally
         static const size_t MIN_BACKOFF_DELAY = 100000;
-        static const size_t MAX_BACKOFF_DELAY = 50000000;
-        static const unsigned MAX_ATTEMPTS      = 10;
+        static const size_t MAX_BACKOFF_DELAY = 500000000;
 
         uint16_t xadd(volatile uint16_t &m, uint16_t v)
             { return ::xadd<uint16_t>(m, v); }
         void atomic_and(volatile uint16_t &m, uint16_t v)
             { ::atomic_and<uint16_t>(m, v); }
 
         volatile uint16_t _rw_lock;
+        unsigned _max_attempts;
```

Comment:
I don't see where _max_attempts is used.

Reply:
I guess it's not clear that this is the comment that caused me to withhold approval. The other comments are questions/suggestions.